src/core_ocean/mode_init/mpas_ocn_init_hurricane.F

```diff
@@ -0,0 +1,342 @@
+
+    !--------------------------------------------------------------------
+    ! Use this section to set initial values
+    !--------------------------------------------------------------------
+
+    block_ptr => domain % blocklist
+    do while(associated(block_ptr))
+       call mpas_pool_get_subpool(block_ptr % structs, 'mesh', meshPool)
+       call mpas_pool_get_subpool(block_ptr % structs, 'state', statePool)
+       call mpas_pool_get_subpool(block_ptr % structs, 'verticalMesh', verticalMeshPool)
+       call mpas_pool_get_subpool(block_ptr % structs, 'diagnostics', diagnosticsPool)
+       call mpas_pool_get_subpool(statePool, 'tracers', tracersPool)
+
+       call mpas_pool_get_dimension(meshPool, 'nCellsSolve', nCellsSolve)
+       call mpas_pool_get_dimension(meshPool, 'nEdgesSolve', nEdgesSolve)
+       call mpas_pool_get_dimension(meshPool, 'nVerticesSolve', nVerticesSolve)
+
+       call mpas_pool_get_dimension(tracersPool, 'index_temperature', index_temperature)
+       call mpas_pool_get_dimension(tracersPool, 'index_salinity', index_salinity)
+
+       call mpas_pool_get_array(meshPool, 'vertCoordMovementWeights', vertCoordMovementWeights)
+       call mpas_pool_get_array(meshPool, 'bottomDepth', bottomDepth)
+       call mpas_pool_get_array(meshPool, 'bottomDepthObserved', bottomDepthObserved)
+       call mpas_pool_get_array(meshPool, 'maxLevelCell', maxLevelCell)
+       call mpas_pool_get_array(meshPool, 'lonCell', lonCell)
+       call mpas_pool_get_array(meshPool, 'latCell', latCell)
+       call mpas_pool_get_array(meshPool, 'latEdge', latEdge)
+       call mpas_pool_get_array(meshPool, 'latVertex', latVertex)
+       call mpas_pool_get_array(meshPool, 'fCell', fCell)
+       call mpas_pool_get_array(meshPool, 'fEdge', fEdge)
+       call mpas_pool_get_array(meshPool, 'fVertex', fVertex)
+
+       call mpas_pool_get_array(tracersPool, 'activeTracers', activeTracers, 1)
+       call mpas_pool_get_array(statePool, 'layerThickness', layerThickness, 1)
+       call mpas_pool_get_array(statePool, 'ssh', ssh, 1)
+
+       call mpas_pool_get_array(meshPool, 'refBottomDepth', refBottomDepth)
+       call mpas_pool_get_array(verticalMeshPool, 'refZMid', refZMid)
+       call mpas_pool_get_array(verticalMeshPool, 'restingThickness', restingThickness)
+
+       do k = 1, nVertLevels
+          refBottomDepth(k) = globalMaxBottomDepth * interfaceLocations(k+1)
+          refZMid(k) = - 0.5_RKIND * (interfaceLocations(k+1) + interfaceLocations(k)) * globalMaxBottomDepth
+       end do
+
+       ! Set vertCoordMovementWeights
+       vertCoordMovementWeights(:) = 1.0_RKIND
+
+       do iCell = 1, nCellsSolve
+
+          ! Set ssh
+          ssh(iCell) = 0.0_RKIND 
+
+          ! enforce minimum and maximum bottom depth
+          bottomDepth(iCell) = max(config_hurricane_min_depth, -1.0_RKIND*bottomDepthObserved(iCell))
+          bottomDepth(iCell) = min(bottomDepth(iCell),globalMaxBottomDepth)
+ 
+          ! Flatten out Bermuda
```

Comment:
Possibly not, but it was giving us trouble for a while. I don't think it hurts to exclude it since we're not resolving it anyway. Let me know if you disagree.

Reply:
We are good for now-- just important to remember we did this for the future.  It shouldn't hurt this run.